src/screens/rewards/components/RewardsStats.tsx

```diff
@@ -70,10 +78,12 @@ export const RewardsStats: React.FC<Props> = ({
                 <RewardsStatsCard
                   key={action.type}
                   title={capitalize(action.type)}
-                  value={action.amount.usd.toLocaleString('en-US', {
-                    style: 'currency',
-                    currency: 'USD',
-                  })}
+                  value={convertAmountToNativeDisplay(
```

Comment:
same here: for the non-fallback USD path, you may be looking for `convertAmountAndPriceToNativeDisplay`: https://github.com/rainbow-me/rainbow/blob/develop/src/helpers/utilities.ts#L272